### PR TITLE
Conan v2 + OS SDK 3.8.0-rc2

### PR DESCRIPTION
**Pull request by jmarrec**

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# Building with conan v2
+
+Check you have `conan >= 2`, and add the `nrel-v2` remote to grab `ruby` and `swig/4.1.1`.
+
+```shell
+conan --version
+conan remote add -f nrel-v2 http://conan.openstudio.net/artifactory/api/conan/conan-v2
+```
```
Comment (jmarrec) [inline BUILDING.md:8]: Step 1

File: BUILDING.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+## Install the conan dependencies into a build folder
+
+```shell
+conan install . --output-folder=../OSApp-build-release --build=missing -c tools.cmake.cmaketoolchain:generator=Ninja -s compiler.cppstd=20 -s build_type=Release
+```
```
Comment (jmarrec) [inline BUILDING.md:14]: Step 2, at root of repo, with conan2

File: BUILDING.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+cmake --preset conan-release [any_futher_configuration_options]
+```
+
+Example:
+
+```
+cmake --preset conan-release \
+  -DQT_INSTALL_DIR:PATH=/opt/Qt/6.6.3/gcc_64 \
+  -DBUILD_PACKAGE:BOOL=ON
+```
```
Comment (jmarrec) [inline BUILDING.md:75]: Step 3, at root still

File: BUILDING.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+Building
+
+```
+cmake --build --preset conan-release
+```
```
Comment (jmarrec) [inline BUILDING.md:81]: Step 4, at root still.

File: .github/workflows/app_build.yml [hunk 5]
```diff
@@ -451,52 +445,46 @@ jobs:
...
+
+          ls $(pwd)/build/OpenStudio-3.8.0 || true
+          OS_DIR=$(find "$(pwd)/build/OpenStudio-3.8.0" -name openstudioConfig.cmake)
+          OS_DIR=$(dirname $OS_DIR)
+          cmake --preset conan-release -DQT_INSTALL_DIR:PATH=${{ env.QT_INSTALL_DIR }} \
+              -DBUILD_DOCUMENTATION:BOOL=${{ env.BUILD_DOCUMENTATION }} \
+              -DBUILD_PACKAGE:BOOL=${{ env.BUILD_PACKAGE }} \
+              -DANALYTICS_API_SECRET:STRING=${{ secrets.ANALYTICS_API_SECRET }} \
+              -DANALYTICS_MEASUREMENT_ID:STRING=${{ secrets.ANALYTICS_MEASUREMENT_ID }} -Dopenstudio_DIR:PATH=$OS_DIR
+        fi
```
Comment (jmarrec) [inline .github/workflows/app_build.yml:478]: Need to remove that block soon. OS SDK 3.8.0-RC2 has a different extracted folder (different SHA) than the github hosted tar.gz and that pwned me real bad.

**Comment by jmarrec** [comment]
Alright, I got all platforms to build but separately, reverted my temp commit that disabled ubuntu and windows, letting a final CI run that hopefully should be all green, and then we can merge this.

**Comment by macumber** [comment]
@jmarrec I built develop with no major issues, thanks for the great instructions!  I was able to load the binaries into SketchUp 2024 which is a first.  Now it is on to debugging the Ruby API changes (apparently no more `exists?` for `File`...).  Thanks!

**Comment by jmarrec** [comment]
Yeah some deprecated stuff was removed in 3.x. File.exists? => File.exist?, tiny stuff like this

**Pull request merged**